gulp/watch.js

```diff
@@ -5,24 +5,25 @@
 */
 
 const gulp = require('gulp')
+
 const config = require('./config.json')
 
 gulp.task('watch-sass', function () {
-  return gulp.watch(config.paths.assets + 'sass/**', {cwd: './'}, ['sass'])
+  return gulp.watch(config.paths.assets + 'sass/**', { cwd: './' }, gulp.series('sass'))
```

Comment:
is `series` necessary if you only have one item?

Reply:
looked into this, `gulp.task('sass')` seems to work too, which reads better to me

https://github.com/gulpjs/gulp/issues/1411#issuecomment-173326435

Reply:
Fair point, it does seem to work fine so I agree going with that approach makes sense. Changes made and pushed.